CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to this project will be documented in this file based on the
 
 ### Bugfixes
 - Function score query: corrected the `score_method` `average` to `avg` #975 
+- Set `json_decode()` assoc parameter to true in `Elastica\Response` #1005
```

Comment:
Sorry to bring this up again. But isn't the affect here, that if BIG_INT is enabled or not could be that the response object looks different?